### DISCUSSION TITLE
Status of v2 Branch?

### DISCUSSION THREAD

**Discussion by carwyn** [dolan-in/dgman#90] [Q&A] [answered]
Hello there, I noticed that the master branch has all the commits from the v2 branch on it. Is the v2 branch still being used or is all the development on master now?

**Comment by wildan2711** [comment, marked as answer]
Hi @carwyn, all the development should be in the master branch now. v2 is in the alpha stage. I was planning to release v2 soon, though I didn't have time these couple of months.
